controllers/packetcluster_controller.go

```diff
@@ -180,7 +179,7 @@ func (r *PacketClusterReconciler) SetupWithManager(ctx context.Context, mgr ctrl
 		WithEventFilter(predicates.ResourceNotPausedAndHasFilterLabel(log, r.WatchFilterValue)).
 		WithEventFilter(predicates.ResourceIsNotExternallyManaged(log)).
 		Watches(
-			&source.Kind{Type: &clusterv1.Cluster{}},
+			&clusterv1.Cluster{},
```

Comment:
CAPDO took a different approach in the cluster_controller Watch from what it is here: 
https://github.com/kubernetes-sigs/cluster-api-provider-digitalocean/pull/527/files#diff-5e0592ebe68415a0d4f801edc0dc467b6610feda7fbfe1ae477ed8f58f3d8fafR66

Their approach to metal_controller Watches matches what is done in this PR.

Reply:
It would be helpful to understand and note what effect this has

Reply:
CAPD is not using `source.Kind`:
https://github.com/kubernetes-sigs/cluster-api/blob/1eb9be94160ff552845dfa651e47014d71bc24f7/test/infrastructure/docker/internal/controllers/dockercluster_controller.go#L206

Reply:
From what I'm seeing here, CAPD and CAPP (if approved) would be setting the pause/unpause watch inside the NewControllerMangedBy call, where CAPDO, Hetzner, and Azure (the three I checked) set up a .Build and then separate add the watch to that builder. Seems like the latter is only useful if you want to do more logic on the builder that we're not doing. But please let me know if I'm overlooking something.